notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>..."AI Explanations returns your images, highlighting the pixels or regions of your image that are most influential to your model's prediction."</p><p> </p><p>(The "pixels or regions" part is important, because IG highlights pixels and XRAI highlights regions)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Thanks for adding "pixels or regions", but could you apply the exact wording I provided?</p>